Makefile

```diff
@@ -36,7 +36,7 @@ BUILD_OS := $(shell uname -s)
 
 # Set the version information for the Kubernetes servers, and build localkube statically
 K8S_VERSION_LDFLAGS := $(shell $(PYTHON) hack/get_k8s_version.py 2>&1)
-MINIKUBE_LDFLAGS := -X k8s.io/minikube/pkg/version.version=$(VERSION)
+MINIKUBE_LDFLAGS := $(K8S_VERSION_LDFLAGS) -X k8s.io/minikube/pkg/version.version=$(VERSION)
```

Comment:
Oops, I think this is actually problematic. LOCALKUBE_LDFLAGS already embeds MINIKUBE_LDFLAGS. I'm not sure if this is why, but when I built with your patch I got a panic:

```
==> /var/lib/localkube/localkube.out <==
unexpected fault address 0x8499600
fatal error: fault
[signal 0xb code=0x1 addr=0x8499600 pc=0x4625fc]

goroutine 1 [running, locked to thread]:
runtime.throw(0x38c90f0, 0x5)
	/usr/local/go/src/runtime/panic.go:547 +0x90 fp=0xc8200e1938 sp=0xc8200e1920
runtime.sigpanic()
	/usr/local/go/src/runtime/sigpanic_unix.go:27 +0x2ab fp=0xc8200e1988 sp=0xc8200e1938
runtime.memmove(0xc82014fd3c, 0x8499600, 0x6)
	/usr/local/go/src/runtime/memmove_amd64.s:136 +0x36c fp=0xc8200e1990 sp=0xc8200e1988
bytes.(*Buffer).WriteString(0xc82014fd18, 0x8499600, 0x6, 0x40, 0x0, 0x0)
	/usr/local/go/src/bytes/buffer.go:144 +0xd1 fp=0xc8200e19f8 sp=0xc8200e1990
k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus.(*MetricVec).hashLabelValues(0xc82014fce0, 0xc8200e1d80, 0x9, 0x9, 0x2925440, 0x0, 0x0)
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus/vec.go:214 +0x169 fp=0xc8200e1ab0 sp=0xc8200e19f8
k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus.(*MetricVec).GetMetricWithLabelValues(0xc82014fce0, 0xc8200e1d80, 0x9, 0x9, 0x0, 0x0, 0x0, 0x0)
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus/vec.go:86 +0xb7 fp=0xc8200e1b10 sp=0xc8200e1ab0
k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus.(*MetricVec).WithLabelValues(0xc82014fce0, 0xc8200e1d80, 0x9, 0x9, 0x0, 0x0)
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus/vec.go:124 +0x58 fp=0xc8200e1b78 sp=0xc8200e1b10
k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus.(*GaugeVec).WithLabelValues(0xc82014fce0, 0xc8200e1d80, 0x9, 0x9, 0x0, 0x0)
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/prometheus/client_golang/prometheus/gauge.go:115 +0x5d fp=0xc8200e1bd0 sp=0xc8200e1b78
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/version.init.1()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/version/version.go:73 +0x328 fp=0xc8200e1e18 sp=0xc8200e1bd0
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/version.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/version/version.go:76 +0x5e fp=0xc8200e1e20 sp=0xc8200e1e18
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/restclient.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/restclient/versions.go:88 +0xac fp=0xc8200e1ec8 sp=0xc8200e1e20
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/autoscaling/unversioned.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/autoscaling/unversioned/horizontalpodautoscaler.go:150 +0x45 fp=0xc8200e1ed0 sp=0xc8200e1ec8
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/import_known_versions.go:39 +0x44 fp=0xc8200e1ed8 sp=0xc8200e1ed0
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/volume.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/volume/volume_linux.go:81 +0x67 fp=0xc8200e1ee0 sp=0xc8200e1ed8
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/aws.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/aws/sets_ippermissions.go:146 +0xce fp=0xc8200e1f18 sp=0xc8200e1ee0
k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/cloudprovider/providers.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/providers.go:28 +0x3b fp=0xc8200e1f20 sp=0xc8200e1f18
k8s.io/minikube/vendor/k8s.io/kubernetes/cmd/kube-apiserver/app.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:283 +0x3f fp=0xc8200e1f28 sp=0xc8200e1f20
k8s.io/minikube/pkg/localkube.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/pkg/localkube/servers.go:74 +0x44 fp=0xc8200e1f30 sp=0xc8200e1f28
k8s.io/minikube/cmd/localkube/cmd.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/cmd/localkube/cmd/start.go:116 +0x45 fp=0xc8200e1f38 sp=0xc8200e1f30
main.init()
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/cmd/localkube/main.go:36 +0x45 fp=0xc8200e1f40 sp=0xc8200e1f38
runtime.main()
	/usr/local/go/src/runtime/proc.go:177 +0x27f fp=0xc8200e1f90 sp=0xc8200e1f40
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1 fp=0xc8200e1f98 sp=0xc8200e1f90

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 5 [runnable]:
k8s.io/minikube/vendor/github.com/golang/glog.(*loggingT).flushDaemon(0x5869920)
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/golang/glog/glog.go:878
created by k8s.io/minikube/vendor/github.com/golang/glog.init.1
	/usr/local/google/home/dlorenc/go/src/k8s.io/minikube/_gopath/src/k8s.io/minikube/vendor/github.com/golang/glog/glog.go:410 +0x297
```